eZ/Publish/API/Repository/Tests/UserServiceAuthorizationTest.php

```diff
@@ -44,6 +44,31 @@ public function testLoadUserGroupThrowsUnauthorizedException()
         /* END: Use Case */
     }
 
+    /**
+     * Test for the loadUserGroupByRemoteId() method.
+     *
+     * @covers \eZ\Publish\API\Repository\UserService::loadUserGroupByRemoteId()
+     * @depends eZ\Publish\API\Repository\Tests\UserServiceTest::testLoadUserGroupByRemoteId
+     */
+    public function testLoadUserGroupByRemoteIdThrowsUnauthorizedException(): void
+    {
+        $this->expectException(\eZ\Publish\API\Repository\Exceptions\UnauthorizedException::class);
```

Comment:
Import can be used and this could be moved just before we expect an exception

Reply:
Fixed in 6f9d052